nominees/un-biodiversity-lab.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "UN Biodiversity Lab",
+  "aliases": [
+    "UNBL"
+  ],
+  "description": "The UN Biodiversity Lab (UNBL) provides access to global data and analytics on people and planet in new ways to empower governments and stakeholders to take action to achieve their commitments to the Convention on Biological Diversity and 2030 Agenda for Sustainable Development. ",
```

Comment:
It's not clear what the application being submitted is and how it relates to the SDGs. There is not any documentation in the listed repositories either.

Reply:
@jwflory I have contacted UNBL on this, shall revert once we hear from them. Thanks.

Reply:
@jwflory @conradsp for now UNBL will be nominated as an open data project. Therefore, the submission will reflect the above change from now. Thanks.

Reply:
Hi @nathanbaleeta, @prajectory and team. I trust you are well. I'm a bit confused here. I understand the inability to accredit as a software given criteria #4 is not met. (Reading the discussion and input - thanks for being thorough - in the other thread on UNBL.) But this should be ready for accreditation as a data DPG, where it is fulfilling all criteria, no? This was the expectation from our end after our last conversation, so not sure why we are at a nomination-phase still.

Reply:
@benbertelsen Sure with some additional information from the UNBL team, the digital solution should be ready soon as a open data DPG. Thanks.

Reply:
Hi @nathanbaleeta and team, it's great to hear that UNBL will be accredited as an open data DPG soon. Just for clarification what extra information will you require from UNBL to fulfill this? Thanks

Reply:
Hi @davidnjagah below are 2 areas we need additional information to update the submission into an open data project:

- Open licensing.  Could you please provide a link that explicitly mentions the open data license used by UNBL (We have tried to scan the [website](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Funbiodiversitylab.org%2F&data=05%7C01%7Cbenjamin.bertelsen%40undp.org%7Cd6760634a0cc40b5d91208da79f42087%7Cb3e5db5e2944483799f57488ace54319%7C0%7C0%7C637956386680226540%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OLdMEeeRXgZDefALWcXdA21qI0LhJdLGTecqZ5gTfXg%3D&reserved=0) but not yet successful)

- Spatial data repository. If available, please share a link to the spatial repository where a user can download data into non-proprietary formats.

Reply:
Great thanks, @nathanbaleeta.